packages/terra-responsive-element/src/ResponsiveElement.jsx

```diff
@@ -14,11 +14,11 @@ const propTypes = {
    */
   children: PropTypes.node,
   /**
-   * A callback function invoked when the breakpoint size changes. onChange(<string>breakpoint)
+   * A callback function invoked when the breakpoint size changes. onChange(<string>breakpoint</string>)
```

Comment:
I imagine these were intending to describe the intended parameter type. Does the props table parse this incorrectly?

Reply:
> Does the props table parse this incorrectly?

Yes, when we type `string` as a tag. Then the MDX loader looks for a closing tag for it as well and if it is not present the loader breaks at `propstableloader` And when I saw the dev site props table It doesn't display the prop type. So, now instead of removing it, I just added the closing tag.

Reply:
You could try this:
```suggestion
   * A callback function invoked when the breakpoint size changes. ```onChange(<string>breakpoint)```
```

Reply:
done here https://github.com/cerner/terra-core/pull/2864/commits/ce59f8d71b6e611399d9e011374024bb4d70de52.